cves/CVE-2015-0220.yml

```diff
@@ -140,8 +164,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    XSS test could've discovered this issue by hitting a website using this
```

Comment:
Grammatical correction, you keep switching between using "XSS" and "xss" and "cross-site scripting", my suggestion would be to settle on one and stay consistent.